src/pytest/test_debug_commands.py

```diff
@@ -197,6 +197,7 @@ def testLogAssert(self):
         self.env.expect('FT.DEBUG', 'LogAssert').error()
         logfile = open(self.env.logDir + '/' + logfile_name, "r")
         for line in logfile:
-            if (str(line).__contains__('(7 == 42) failed on /home/ariel/redis/RediSearch/src/debug_commads.c:LogAssert')):
+            if (str(line).__contains__('False: 7 == 42')):
+                print line
```

Comment:
@ashtul I do not understand how come rltest does not failed this test because the redis crashes here. Regardless this test purpose was to verify that we are getting the crash report, now that we are use RM_Assert we can trust redis to print the crash report for sure and we no longer need this test. What do you think?

Reply:
@MeirShpilraien Yes, let's remove. RLTest gives the whole dump info and it is confusing.
Should I remove the FT.DEBUG command as well?

Reply:
Yes good idea